contracts/VestingHandler.sol

```diff
@@ -16,60 +17,78 @@ contract VestingHandler is TokenHandler {
     *   version2 : release(address token)
     *
     *   version type has to be passed in to complete the release, default is version1.
-    *  0 => version1
-    *  1 => version2
+    *   0 => version1
+    *   1 => version2
     */
 
     enum vestingContractVersion { v1, v2 }
 
     address public vestingContract;
     vestingContractVersion public targetVersion;
 
-    constructor ( vestingContractVersion _targetVersion, address _vestingContract, address _targetToken ) public
-    TokenHandler(_targetToken){
+    constructor(
+        vestingContractVersion _targetVersion,
+        address _vestingContract,
+        address _targetToken
+    )   public
+        TokenHandler(_targetToken)
+    {
+        require(_vestingContract != address(0x0), "Please specifity address of vesting contract!");
```

Comment:
Same here, we want to allow the handler be deployed before or after the Vesting contract, should the address be needed when deploying the vesting contract, which is very common